pom.xml

```diff
@@ -892,9 +935,10 @@
             351
         </noSnapshotScala213.buildvers>
         <snapshotScala213.buildvers>
+            400
```

Comment:
this will fail our nightly build. please remove for now until it could be built correctly

Reply:
Are we plan to aggregate spark400 snapshot jar into the final dist jar?

For the nightly build CI, as all other shims are build against JDK8, if we also aggregate 400 shim against JDK17 into the final dist jar, there should be some class files confections due to building with diff JDK.

Reply:
@razajafri can we remove this before we have a solution for the nightly build? thanks

Reply:
We plan on fixing the nightly builds before this goes in.